examples/datadog-express-apm/container-agent/src/env.sample

```diff
@@ -0,0 +1,29 @@
+# commercetools env
+CTP_PROJECT_KEY=
+CTP_CLIENT_ID=
+CTP_CLIENT_SECRET=
+CTP_AUTH_URL=https://auth.<your-region>.gcp.commercetools.com
+CTP_API_URL=https://api.<your-region>.gcp.commercetools.com
+# CTP_CLIENT_USERNAME=
+# CTP_CLIENT_PASSWORD=
+
+# datadog envs
+DD_ENV=dev
+DD_LOGS_INJECTION=true
+DD_TAGS="env:${DD_ENV}"
+DD_PROCESS_AGENT_ENABLED=true
+DD_DOGSTATSD_NON_LOCAL_TRAFFIC=true # if expecting external metrics
+DD_APM_NON_LOCAL_TRAFFIC=true
+DD_APM_ENABLED=true
+DD_APM_ANALYZED_SPANS='api-gateway|express.request=1,auth|express.request=1,user|express.request=1'
+DD_BIND_HOST=0.0.0.0
+DD_LOGS_ENABLED=true
+DD_API_KEY=<datadog-api-key>
+DD_APM_DD_URL=<https://xxx.datadoghq.xxx>
```

Comment:
I think we should use the correct DD_URL here, you can add a comment to specify the need to use the correct region.

Reply:
Added a detailed comment explaining how to find the correct region-specific Datadog URL, just as clear guidance to users without cluttering the `.env.sample` file with region-specific information.
As example of Java PR about regions